nano/node/bootstrap/bootstrap_ascending.cpp

```diff
@@ -112,6 +146,10 @@ void nano::bootstrap_ascending::buffered_iterator::fill ()
 		{
 			buffer.push_back (*pending_iterator);
 		}
+		else
+		{
+			warmup_m = false;
```

Comment:
this can be true only for the first time the fill() function is called. Is that right?

Reply:
This should be true until after the iterator has gone through both the account and pending tables once.